### DISCUSSION TITLE
Courseplay error

### DISCUSSION THREAD

**Discussion by BRADP1980** [Courseplay/Courseplay_FS22#1810] [Q&A] [answered]
Hello. I downloaded the new Courseplay 7.1.0.1 and moved it into my mods folder for PC and when i start my game i get an error that tells me this saved game uses downloadable content which is not installed. i Continue the game to be sure its checked in my mods and its not there at all. i can reload 7.1.0.0 and no issues loading. any ideas? Also i am running the newest Version on the game as well. I've uploaded a screen shot also. Thanks!
![Screenshot (3)](https://user-images.githubusercontent.com/107006199/181165078-d82aea81-5ee9-4ec8-b56f-a52cdf5c79a5.png)

**Comment by BalticHawk** [comment]
Did you try to delete the file and download it again?
Could be a damaged file or a mistake while you moved it.
a log file could give some answers as well.

**Reply by BRADP1980** [reply]
Yes i deleted the old a redownloaded the new several times still with no luck.  This is what i see at the start of the log. If you need more please let me know. thank you for the help.
![Screenshot (5)](https://user-images.githubusercontent.com/107006199/181237513-57f8468c-99ed-4d5f-8701-a1ccb55e6cb8.png)

**Comment by Tensuko** [comment]
Logfile....
100% sure it is either (2) becouse you dirct download into your mods folder or anything else.
No log, not solution...

**Reply by BRADP1980** [reply]
Sorry for the misunderstanding. i downloaded the file into my download folder then after download finished i moved it into the mods folder. 
This is what i see at the start of the log. If you need more please let me know. thank you for the help.  Wasn't sure if you meant the full log file.
![Screenshot (5)](https://user-images.githubusercontent.com/107006199/181237928-30106b0a-cade-42ac-81a4-98cbc00c5b3b.png)

**Reply by BalticHawk** [reply]
The full logfile would be grate. And no copy and paste. Just drag and drop the file. :)

**Reply by BRADP1980** [reply]
i feel silly but it is up and running now. i could of swore i had already been running FS 22 V1.6 but i guess i wasnt. i am now and all seems good. Thank you so much for all your help! :)

**Comment by Tensuko** [comment, marked as answer]
Might be wrong/old FS Version.
Actual one is 1.6!

**Reply by BRADP1980** [reply]
That seem to have worked. i feel so silly i could of swore i had already moved to 1.6 lol. i have now got 1.6 running and all mods are working. even the ones i had downloaded and never worked before. Thank you for all your help! :)
